contracts/Gateway.sol

```diff
@@ -90,6 +94,8 @@ contract Gateway is ProtocolVersioned, Owned {
         WorkersInterface _workers,
         uint256 _bounty,
         bytes32 _uuid,
+        uint256 _conversionRate,
+        uint8 _conversionRateDecimals,
         address _openSTProtocol)
```

Comment:
All the code we touch now should probably adhere to the [Solidity Style Guide](https://github.com/OpenSTFoundation/openst-protocol/blob/develop/SOLIDITY_STYLE_GUIDE.md#function-declarations). Please check how function declarations should be formatted and update all functions.